app/reducers/ui.js

```diff
@@ -35,6 +35,9 @@ export default function auth(state = initialState, action) {
     case 'FETCHING_UPDATED_TASK_INFORMATION_SUCCESS': {
       return Object.assign({}, state, { showTaskDetailsSidebarLoading: false });
     }
+    case 'TOGGLE_SWIMLANE': {
+      return document.querySelector('section[data-reactid*="'+action.payload.id+'"]').classList.toggle('swimlane--collapsed');
```

Comment:
I originally had a property called swimlaneCollapsed that did this. The trouble I had was targeting specific lanes - I could expand/collapse all lanes, but not just one. I'll look at the docs you linked and keep trying :-)